projects/packages/status/src/class-modules.php

```diff
@@ -363,10 +363,6 @@ public function activate( $module, $exit = true, $redirect = true ) {
 			return false;
 		}
 
-		if ( ! $this->is_module( $module ) ) {
```

Comment:
Removing this entirely feels like a loose move, I think it's just meant to be defensive to prevent errors. Could move this in Jetpack check below instead.

Reply:
Moved it in c1e685ab04